template/README.rst

```diff
@@ -0,0 +1,40 @@
+<demo_name> Demo
+=============
+Short blurb on what this demo does.
+
+Usage
+-----
+A simple command on how to run the demo. For example,
+::
```

Comment:
"A simple command on how to run the demo"
Do you mean a "simple description" or "A command that runs"? I mean a "command runs" (without how to) and do you require it be simple